firewall.go

```diff
@@ -151,6 +151,31 @@ func (c *Client) ListFirewallRules(id string) ([]FirewallRule, error) {
 	return firewallRule, nil
 }
 
+// FindFirewallRules finds a firewall Rule by either part of the ID
+func (c *Client) FindFirewallRules(idFirewall string, search string) (*FirewallRule, error) {
```

Comment:
finds **a** firewall Rule so it should be `FindFirewallRule` not `FindFirewallRules`. Also this should only take a single parameter of `search string`

Reply:
Also the comment says "by either part of the ID" and then finishes, it should say "or by name" and the function body should also search for part of the name. Using just the ID doesn't match the other similar functions.

Reply:
@andyjeffries the `idFirewall` is because the `FindFirewallRules`  use `ListFirewallRules` inside who list all rules inside the firewall

Reply:
Sorry, you're not understanding. You could just use search and look for either field like:

https://github.com/civo/civogo/blob/f149ccdce08e326133fe16513b3e499303fe6fee/network.go#L91

Reply:
`FirewallRules` don't have name

Reply:
Ohhhh, that's a good point. Keep the parameter name consistent with the other functions then `search` and remove the `idFirewall` parameter.